apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -45,7 +45,7 @@ defmodule Aecore.Chain.BlockValidation do
     end
   end
 
-  @spec single_validate_block!(Block.t()) :: {:error, term()} | :ok
+  @spec single_validate_block!(Block.t) :: {:error, term()} | :ok
```

Comment:
So are you keeping the erlang-style parens on the types or not? 😩 
Some types have parens, some don't. I would greatly encourage consistency here, and sticking with it.

Reply:
@d-velev @cytadela8 havent we decided to keep the erlang-style parens?